cmd/nerdctl/compose_exec.go

```diff
@@ -69,7 +60,7 @@ func composeExecAction(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	tty, err := cmd.Flags().GetBool("tty")
+	noTty, err := cmd.Flags().GetBool("no-TTY")
```

Comment:
Shouldn't we still check `--tty` too?

It is hidden in Docker Compose v2, but it still exists
https://github.com/docker/compose/blob/b1a26dac1d3dc201d0d026cc16cb9baadaacffd7/cmd/compose/exec.go#L77

Reply:
HI @AkihiroSuda 

Thanks for the PR review.

When using `docker compose exec -t=true --no-TTY`, the docker do not check the `--tty`.

```
root@kay200:/tmp# docker compose -f a.yaml exec --tty=true --no-TTY  svc0 stty
stty: standard input: Not a tty
```

So it be removed to keep compatibility :-)

Reply:
👀 Could you add a code comment to explain that behavior?

Reply:
Thanks 
There are comments on it :-)